embl_robot.py

```diff
@@ -296,10 +296,13 @@ def mount(self, gov_robot, puckPos,pinPos,sampID,**kwargs):
           logger.error(e)
           e_s = str(e)
           if (e_s.find("Fatal") != -1):
-            daq_macros.robotOff()
-            daq_macros.disableMount()
-            daq_lib.gui_message(e_s + ". FATAL ROBOT ERROR - CALL STAFF! robotOff() executed.")
-            return MOUNT_FAILURE
+            if self.checkIcedPin(e_s):
```

Comment:
this sounds like this is a correctly-mounted check, with the ability to handle iced pin, instead of only checking for an iced pin. the name also does not suggest a return type, while using something like `isIcedPin()` suggests a boolean.
how about something like `isSampleDetected()` for the function name?

Reply:
Fixed in [62cdacb](https://github.com/NSLS-II/lsdc/pull/372/commits/62cdacbcc688a882171e28100a5c682093e870ba)